app/src/main/res/layout/card_what_to_do_leitfaden.xml

```diff
@@ -141,6 +141,82 @@
 					android:orientation="vertical"
 					app:layout_constraintTop_toBottomOf="@id/card_encounters_last_call">
 
+					<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
The two XML layouts seem to have a lot of duplicate layout parts. Could be shared with a common included layout.

Reply:
Yes! I have merged this duplicate layout code.